bin/torii/src/main.rs

```diff
@@ -145,7 +145,7 @@ async fn main() -> anyhow::Result<()> {
     // Get world address
     let world = WorldContractReader::new(args.world_address, &provider);
 
-    let mut db = Sql::new(pool.clone(), args.world_address).await?;
+    let db = Arc::new(RwLock::new(Sql::new(pool.clone(), args.world_address).await?));
```

Comment:
Do we need a RWLock here? I think we should be able to let SQLite manage the locking and create new db connections instead?